internal/clientcache/internal/daemon/options.go

```diff
@@ -108,3 +109,10 @@ func WithBoundaryTokenReaderFunc(_ context.Context, fn cache.BoundaryTokenReader
 		return nil
 	}
 }
+
+func WithForceResetSchema(_ context.Context, force bool) Option {
```

Comment:
Can you add doc for this function? And a test?

Reply:
Done!